packages/components/src/components/carousel/carousel.stories.ts

```diff
@@ -54,89 +61,135 @@ export const Default = {
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: { type: 'attribute', name: 'variant' },
-        y: { type: 'attribute', name: 'inverted' }
-      },
-      args,
-      constants: [{ type: 'attribute', name: 'autoplay', value: true }],
-      options: {
-        templateBackgrounds: { alternate: 'y', colors: ['rgb(var(--sd-color-primary, 0 53 142))', 'white'] }
-      }
-    });
-  }
+  name: 'Inverted',
+  render: () => html`
+    <div class="bg-primary p-8">
+      <sd-carousel inverted>
+        <sd-carousel-item>
+          <div class="slot slot--border slot--text h-16">Default slot 1</div>
+        </sd-carousel-item>
+        <sd-carousel-item>
+          <div class="slot slot--border slot--text h-16">Default slot 2</div>
+        </sd-carousel-item>
+        <sd-carousel-item>
+          <div class="slot slot--border slot--text h-16">Default slot 3</div>
+        </sd-carousel-item>
+      <sd-carousel>
+    </div>
+  `
 };
 
 /**
  * Use the `loop` attribute to enable/disable the looping of your slides.
  */
 
 export const Loop = {
-  parameters: { controls: { exclude: 'loop' } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: { type: 'attribute', name: 'loop' }
-      },
-      args
-    });
-  }
+  name: 'Loop',
+  render: () => html`
+    <div>
+      <sd-carousel loop>
+        <sd-carousel-item>
+          <div class="slot slot--border slot--text h-16">Default slot 1</div>
+        </sd-carousel-item>
+        <sd-carousel-item>
+          <div class="slot slot--border slot--text h-16">Default slot 2</div>
+        </sd-carousel-item>
+        <sd-carousel-item>
+          <div class="slot slot--border slot--text h-16">Default slot 3</div>
+        </sd-carousel-item>
+      <sd-carousel>
+    </div>
+  `
 };
 
 /**
  * Use the `autoplay` attribute to toggle autoplay.
  */
 
 export const Autoplay = {
-  parameters: { controls: { exclude: 'autoplay' } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: { type: 'attribute', name: 'autoplay' }
-      },
-      constants: [
-        { type: 'attribute', name: 'variant', value: 'dot' },
-        { type: 'attribute', name: 'loop', value: 'true' }
-      ],
-      args
-    });
-  }
+  name: 'Autoplay',
+  render: () => html`
+    <div>
+      <sd-carousel autoplay>
+        <sd-carousel-item>
+          <div class="slot slot--border slot--text h-16">Default slot 1</div>
+        </sd-carousel-item>
+        <sd-carousel-item>
+          <div class="slot slot--border slot--text h-16">Default slot 2</div>
+        </sd-carousel-item>
+        <sd-carousel-item>
+          <div class="slot slot--border slot--text h-16">Default slot 3</div>
+        </sd-carousel-item>
+      <sd-carousel>
+    </div>
+  `
 };
 
 /**
  * Use the `slides-per-page` attribute to set the number of slides that would be shown at a given time.
```

Comment:
Use the `slides-per-page` attribute to set the number of slides displayed at once

Reply:
Updated